docs/advanced/picking.md

```diff
@@ -73,12 +73,36 @@ calculatePickingColors(attribute) {
 
 ## Custom Shaders
 
-Deck.gl layers use luma.gl's [`Picking Module`](http://uber.github.io/luma.gl/#/documentation/api-reference/shader-module). If you are writing custom shaders for your layers, please refer to above luma.gl documentation on how to add picking logic to your shaders. Also make sure to create `Model` object with `picking` module. ()
+All core layers (including composite layers) support picking using luma.gl's `picking module`. If you are using custom shaders with any of the core layers or building custom layers with your own shaders following steps are needed to achieve `Picking`.
```

Comment:
* Feels like what follows is a subsection in custom shaders, specifically about how to implement picking. Maybe add a sub header?

Reply:
This whole section should only talk about custom shaders, just wanted specify that core layer's shader already are using `picking` module. And any custom shader need to do the same.